pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/RelToStageConverter.java

```diff
@@ -77,11 +79,17 @@ public static StageNode toStageNode(RelNode node, int currentStageId) {
       return convertLogicalAggregate((LogicalAggregate) node, currentStageId);
     } else if (node instanceof LogicalSort) {
       return convertLogicalSort((LogicalSort) node, currentStageId);
+    } else if (node instanceof LogicalWindow) {
```

Comment:
I ran the following query and tried to see what `relRoot` in `StagePlanner.makePlan` look like.
`select playerID, avg(runs) over(partition by playerID) from baseballStats limit 10 OPTION(useMultistageEngine=true)`
For this query, relRoot has:
```
LogicalSort
  LogicalProject
     exps (CAST(/(SUM($26) OVER (PARTITION BY $27), COUNT($26) OVER (PARTITION BY $27))):INTEGER NOT NULL)
     LogicalTableScan
```
`LogicalWindow` does not seem to appear in the relRoot. Also avg(runs) over(partition by playerID) got replaced by cast, sum, and count functions? I am wondering if we can avoid such rewrites?